backend/vcdat/GraphicsMethods.py

```diff
@@ -4,7 +4,7 @@
 _ = vcs.init()
 _methods = {}
 _2d_methods = ('scatter', 'vector', 'xvsy', 'stream', 'glyph', '3d_vector', '3d_dual_scalar')
-_primitives = ('line', 'marker', 'fillarea', 'text')
+_primitives = ('line', 'marker', 'fillarea', 'text','isofill')
```

Comment:
Does this effect what is returned to the app? I ask because this change is outside the testing file thus when the app calls this function there is a new result returned.